### PR TITLE
fix: handle arc4.Address.native

### PR DESCRIPTION
**Pull request by daniel-makerx**

### CODE REVIEW

File: test_cases/arc4_types/address.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+
+class Arc4AddressContract(Contract):
+    def approval_program(self) -> bool:
+        address = arc4.Address(Txn.sender)
+        assert address == Txn.sender
+        assert address.length == 32
+        assert address.native == Txn.sender
+
+        zero_address = arc4.Address(Global.zero_address.bytes)
```
Comment (tristanmenzel) [inline test_cases/arc4_types/address.py:13]: I think we can do `arc4.Address()` now right?

Comment (tristanmenzel) [reply test_cases/arc4_types/address.py:13]: (to get a zero address)

Comment (daniel-makerx) [reply test_cases/arc4_types/address.py:13]: Yeah, but in this case I wanted to cover the constructor case that takes a bytes, `Global.zero_address` was just a convenient source of another address

Comment (tristanmenzel) [reply test_cases/arc4_types/address.py:13]: ah ok, I just use `Txn.sender.bytes` :)

**Review by tristanmenzel — APPROVED** [review]

**Pull request merged**